templates/AureliaSpa/wwwroot/web.config

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't think this file is needed. It's not present in any of the other templates.